paypal/express/gateway.py

```diff
@@ -206,6 +206,8 @@ def set_txn(basket, shipping_methods, currency, return_url, cancel_url, update_u
         params['L_PAYMENTREQUEST_0_AMT%d' % index] = _format_currency(
             line.unit_price_incl_tax)
         params['L_PAYMENTREQUEST_0_QTY%d' % index] = line.quantity
+        params['L_PAYMENTREQUEST_0_ITEMCATEGORY%d' % index] = (
+            'Physical' if product.is_shipping_required else 'Digital')
```

Comment:
This product type determination is not quite reliable. For instance, won't be true for "pickup" shipping.

Reply:
@dorianfm what do you think?

Reply:
There are the two possible values of the api: See [the api reference](
https://developer.paypal.com/docs/classic/express-checkout/integration-guide/ECRecurringPayments/?mark=itemcategory#identify-items-as-digital-or-physical-goods).

Reply:
This seems fine to me.

I agree with @ddiazpinto here - the physical / digital is not a distinction of shipping method with PayPal, rather of product category, and IIRC it's only used by PayPal to discern whether you should get a discounted transaction rate for digital only sales. 

If a developer wanted to do a 'pickup' sale that would be handled by a zero priced Shipping Method (in Oscar's `shipping.methods`), that could be supplied along side the paid options by Oscar's `shipping.repository.get_available_shipping_methods()`.

Reply:
Ok, agreed.